packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/user-pool-group-stack-transform.ts

```diff
@@ -229,23 +257,23 @@ export class AmplifyUserPoolGroupTransform extends AmplifyCategoryTransform {
     return cfnUserPoolGroupStack;
   };
 
-  public saveBuildFiles = async (context: $TSContext, template: Template): Promise<void> => {
+  public saveBuildFiles = async (__context: $TSContext, template: Template): Promise<void> => {
```

Comment:
nit: it's probably better to just remove the context from signature

Reply:
it has couplings with other pieces of code , so I have disabled it now